lib/axiom/fs/stream/stub_file_system.test.js

```diff
@@ -328,7 +328,7 @@ describe('StubFileSystem', function () {
             expect(writeResult).toBeDefined();
             expect(writeResult.dataType).toBe(DataType.Value);
             // TODO(rpaquay): Shoud this be `data` instead?
-            expect(writeResult.data).toBe(null);
+            expect(writeResult.data).toBe('This is another text file');
```

Comment:
nit: remove the "todo" comment and use the `data` variable (versus hard-coding the string value)